vendor/bat-native-ads/src/bat/ads/internal/client.cc

```diff
@@ -366,8 +366,13 @@ void Client::ResetAdsUUIDSeen(
 
 void Client::UpdateNextCheckServeAdTimestampInSeconds() {
   auto timestamp_in_seconds = Time::NowInSeconds();
-  timestamp_in_seconds += base::Time::kSecondsPerHour /
-      ads_client_->GetAdsPerHour();
+
+  if (client_state_->next_check_serve_ad_timestamp_in_seconds == 0) {
+    timestamp_in_seconds += 2 * base::Time::kSecondsPerMinute;
+  } else {
+    timestamp_in_seconds += base::Time::kSecondsPerHour /
+        ads_client_->GetAdsPerHour();
+  }
```

Comment:
Just checking - how this is iOS/Android specific?

Reply:
This call is only called as part of `DeliverNotifications` which is how ads are delivered on mobile. Ads are delivered on desktop using `OnUnidle`